pom.xml

```diff
@@ -1044,8 +1044,7 @@
       <dependency>
         <groupId>com.google.code.gson</groupId>
         <artifactId>gson</artifactId>
-        <version>2.3.1</version>
-        <scope>test</scope>
+        <version>2.9.0</version>
```

Comment:
We need to be careful with this change since it is changing the scope for all modules that are not currently overriding the scope